core/server/middleware/validation/blog-icon.js

```diff
@@ -61,7 +61,7 @@ module.exports = function blogIcon() {
 
         // CASE: file should not be larger than 100kb
         if (!validIconSize(req.file.size)) {
-            return next(new errors.RequestEntityTooLargeError({message: i18n.t('errors.api.icons.fileSizeTooLarge', {extensions: iconExtensions})}));
+            return next(new errors.ValidationError({message: i18n.t('errors.api.icons.invalidFile', {extensions: iconExtensions})}));
```

Comment:
It's possible to show whatever error message we want in the client but in this case I think a `ValidationError` is more appropriate - `RequestEntityTooLarge` should be reserved for when the proxy/web/server configuration stops an upload due to size before it has a chance to be validated.